vulnfetcher/nodeswg/nodeswg.go

```diff
@@ -83,36 +89,52 @@ func (n *Fetcher) Fetch() error {
 
 		return nil
 	})
+	if err != nil {
```

Comment:
92-97 would be equivalent to just `return err`.

Reply:
:+1:  not even in #15, will make a commit for both !

Reply:
pushed !